.github/workflows/ci.yml

```diff
@@ -297,6 +297,62 @@ jobs:
       - shell: bash -l {0}
         run: bash <(curl -s https://codecov.io/bash)
   
+  test-asv-benchmarks:
+    needs: [lint-commit, lint-flake8, lint-black, test-api, test-headers]
+    runs-on: ubuntu-latest
+    env:
+      MODIN_ENGINE: ray
+      MODIN_MEMORY: 1000000000
+      TestDatasetSize: small
```

Comment:
Looks like it is a class name, corresponding env name is `MODIN_TEST_DATASET_SIZE`.

Reply:
Yep, thanks @amyskov !